src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="action-attr"
+    method="post"
+    data-qa="big"
+    class="bar"
+    >
+      <div class="fon fon-big">
+        <img
+        src="images\Search.svg"
+        alt="icon"
+        class="bigIcon"
+        >
+        <input
+        type="text"
+        name="bigbar"
+        id="bigbar"
+        class="input inputBig"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </div>
+    </form>
+    <form
```

Comment:
Add empty lines between multiline sibling blocks of HTML

Reply:
Done